client/src/pages/Map/Map.js

```diff
@@ -85,6 +86,17 @@ export default function Map({
   const queryClient = useQueryClient();
   const selectionEnabledRef = useRef(selectionEnabled);
   const currentFeature = useRef(null);
+  const hasInitialFlyToFired = useRef(false);
```

Comment:
This is not critical right now, but it will be in the future when we add other onLoad flyTo effects such as geolocation. We want to make sure we don't fly to user's initial tree Id and then flies to user's geolocation, etc.